src/interfaces.ts

```diff
@@ -110,6 +113,7 @@ export interface AuthorizedData {
   sessionId: string;
   organizationId: string | null;
   role: string | null;
+  roles: string[] | null;
```

Comment:
Sorry, [broken record here](https://github.com/workos/authkit-react/pull/71#discussion_r2357013270), why not `string[]` like `permissions`, `entitlements`, and `featureFlags`?